packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-## 0.3.10+3
+## 0.3.10+4
 
 * Don't log an unknown setting key error for 'debuggingEnabled' on iOS.
```

Comment:
the wrong logs

Reply:
thanks for noticing, I missed that after rebasing with master.